src/tests/Microsoft.PowerFx.Core.Tests/LexerTests.cs

```diff
@@ -568,5 +568,71 @@ public void TestUnsupportedDecimalSeparatorCausesFallback()
 
             CultureInfo.CurrentCulture = oldCulture;
         }
+
+        [Fact]
+        public void TestStringInterpolation()
+        {
+            Token[] tokens;
+            string value;
+
+            value = "$\"Hello {name}\"";
+            tokens = TexlLexer.LocalizedInstance.LexSource(value);
+            Assert.NotNull(tokens);
+            Assert.Equal(7, tokens.Length);
+            Assert.Equal(TokKind.StrInterpStart, tokens[0].Kind);
+            Assert.Equal(TokKind.StrLit, tokens[1].Kind);
+            Assert.Equal(TokKind.IslandStart, tokens[2].Kind);
+            Assert.Equal(TokKind.Ident, tokens[3].Kind);
+            Assert.Equal(TokKind.IslandEnd, tokens[4].Kind);
+            Assert.Equal(TokKind.StrInterpEnd, tokens[5].Kind);
+            Assert.Equal(TokKind.Eof, tokens[6].Kind);
+
+            value = "$\"Hello {Table({a: 5})} World!\"";
```

Comment:
>           value = "$\"Hello {Table({a: 5})} World!\""; [](http://example.com/codeflow?start=1&length=55)

nit: can you split these into separate [Fact] tests so that the run independently?

Reply:
Done